addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/service/NetworkService.java

```diff
@@ -129,7 +129,7 @@ public void run() {
             try {
                 ReceiveDHCPRequestPackets.register(InetAddress.getByName(hostname).getHostAddress(), stateUpdate);
             } catch (SocketException | UnknownHostException e) {
-                logger.error("Cannot use DHCP listen: " + e.getMessage());
+                logger.error("Cannot use DHCP listen: {}", e.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace